.github/workflows/netlify.yml

```diff
@@ -0,0 +1,51 @@
+name: Build and Deploy to Netlify
+on:
+  push:
+    branches:
+      - "master"
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "pablo/**"
+      - "tropicar/**"
+      - "dapplion/**"
+      - "eduadiaz/**"
+      - "luka/**"
+  pull_request:
+    branches:
+      - "master"
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "pablo/**"
+      - "tropicar/**"
+      - "dapplion/**"
+      - "eduadiaz/**"
+      - "luka/**"
```

Comment:
I would run this on all branches, you can just do 
```
push:
pull_request:
```

Reply:
it's `eduadiez` 😅